galata/test/documentation/data/extensions-search-all.json

```diff
@@ -0,0 +1,202 @@
+          { "username": "zuoyuanh", "email": "david.hzy97@gmail.com" },
+          { "username": "ellisonbg", "email": "ellisonbg@gmail.com" },
+          { "username": "blink1073", "email": "steven.silvester@ieee.org" },
+          { "username": "jasongrout", "email": "jason@jasongrout.org" },
+          { "username": "sylvaincorlay", "email": "sylvain.corlay@gmail.com" },
+          { "username": "ian-r-rose", "email": "ian.r.rose@gmail.com" },
+          { "username": "gnestor", "email": "grantnestor@gmail.com" },
+          { "username": "parente", "email": "parente@gmail.com" },
+          { "username": "mpacer", "email": "mpacer@berkeley.edu" },
+          { "username": "jaipreets", "email": "jaipreet@amazon.com" },
+          { "username": "minrk", "email": "benjaminrk@gmail.com" },
+          { "username": "saulshanabrook", "email": "s.shanabrook@gmail.com" },
+          { "username": "hoo761", "email": "jacobrhoussian@gmail.com" },
+          { "username": "zsailer", "email": "zachsailer@gmail.com" },
+          { "username": "telamonian", "email": "telamonian@hotmail.com" },
+          { "username": "fcollonval", "email": "fcollonval@gmail.com" },
+          { "username": "lresende", "email": "lresende@us.ibm.com" },
+          { "username": "consideratio", "email": "erik.i.sundell@gmail.com" },
+          { "username": "ajbozarth", "email": "ajbozart@us.ibm.com" },
+          { "username": "marthacryan", "email": "martha.cryan326@gmail.com" },
+          { "username": "jtpio", "email": "jeremy@jtp.io" },
+          { "username": "echarles", "email": "eric@datalayer.io" },
+          { "username": "goanpeca", "email": "goanpeca@gmail.com" },
+          { "username": "mbektas", "email": "mbektas@outlook.com" },
+          { "username": "loichuder", "email": "loic.huder@esrf.fr" }
+        ]
+      },
+      "score": {
+        "final": 0.3189719547427941,
+        "detail": {
+          "quality": 0.4344936230693638,
+          "popularity": 0.20559200330090951,
+          "maintenance": 0.3333333333333333
+        }
+      },
+      "searchScore": 9.516456e-8
+    },
+    {
+      "package": {
+        "name": "@jupyterlab/translation",
+        "scope": "jupyterlab",
+        "version": "3.4.4",
+        "description": "JupyterLab - Translation services",
+        "keywords": ["jupyter", "jupyterlab", "jupyterlab-extension"],
+        "date": "2022-07-21T14:51:44.592Z",
+        "links": {
+          "npm": "https://www.npmjs.com/package/%40jupyterlab%2Ftranslation",
+          "homepage": "https://github.com/jupyterlab/jupyterlab",
+          "repository": "https://github.com/jupyterlab/jupyterlab",
+          "bugs": "https://github.com/jupyterlab/jupyterlab/issues"
+        },
+        "author": { "name": "Project Jupyter" },
+        "publisher": {
+          "username": "fcollonval",
+          "email": "fcollonval@gmail.com"
+        },
+        "maintainers": [
+          { "username": "3coins", "email": "pyjain@gmail.com" },
```

Comment:
Maybe we could use placeholder emails here? (even though they are publicly available from npm)

Reply:
Oh yes, I copied the dump without a second thought. Will move to dummy users.

Reply:
:+1: